docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -3917,6 +3918,31 @@ image::docs/images/meraki-splash-config.png[scaledwidth="100%",alt="Splash page
 
 NOTE: Even if you enabled the bridge mode on the access point, it will still send a NATed IP address during the initial HTTP request. To workaround this you will need to forward the DHCP traffic of the network your nodes are connected to so PacketFence has the proper IP information of the devices.
 
+Version 2
++++++++++
+
+NOTE: While using the WebAuth mode on the Meraki controller with a version superior to X.X, you need to use "Role mapping by Switch Role" on the switch configuration. More details below.
+
+In the version X.X of the Meraki firmware, the WebAuth mode has been improved and now allow CoA, and the use of redirect-URL instead of passing by the external captive portal.
+
+The configuration for your SSID should look like this:
+
+image::docs/images/Meraki-config-1.png[scaledwidth="100%",alt="Security and RADIUS configuration"]
+
+image::docs/images/Meraki-config-2.png[scaledwidth="100%",alt="Network configuration"]
+
+NOTE: It is mandatory that you use the Airespace-ACL-Name as "RADIUS attribute specifying group policy name".
+
+While done you will need to add the address for the redirection in the switch configuration in PacketFence. Add the Meraki IP to PacketFence, then choose the module "Meraki cloud controller v2" and add the portal interface ip in the "Portal URL" section. It should look like the following:
+
+----
+ http://192.168.1.5
+----
+
+The switch Role you will enter, will be the one you created as a group policy in the Meraki configration. To do so, go through `Network-wide->Group policies`, here you can create policies that will be applied to the devices which register through this Meraki network.
```

Comment:
Move the role creation explanation above (before the pf switch creation)

Also just rewrite it to the following : 
Next, configure the roles for the devices on your network. Go in 'Network-wide->Group policies', then you will be able to create policies that can be configured in the roles in the switch configuration in PacketFence.